terra_oracle_vote.py

```diff
@@ -24,6 +24,8 @@
 import requests
 from prometheus_client import start_http_server, Summary, Counter, Gauge, Histogram
 import aiohttp
+from pyband.obi import PyObi
+from pyband.client import Client
```

Comment:
requirements.txt Can you add that dependency to that file?

Reply:
done